tools/ci_build/build.py

```diff
@@ -50,6 +50,49 @@ def _check_python_version():
 _check_python_version()
 
 
+def _openvino_verify_device_type(device_read):
```

Comment:
unrelated change?

Reply:
I can split this out if you want. I couldn't take the insanity of defining a random function in the middle of nowhere with a name that provided no context to what it did.

Reply:
https://github.com/microsoft/onnxruntime/pull/5822